pkg/controller/direct/bigquerydataset/bigquerydataset_mappings.go

```diff
@@ -21,314 +21,219 @@
 package bigquerydataset
 
 import (
-	"fmt"
-	"time"
+	"strconv"
 
-	pb "cloud.google.com/go/bigquery"
 	krm "github.com/GoogleCloudPlatform/k8s-config-connector/apis/bigquery/v1beta1"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/controller/direct"
+	api "google.golang.org/api/bigquery/v2"
 )
 
-func Access_FromProto(mapCtx *direct.MapContext, in *pb.AccessEntry) *krm.Access {
+func BigQueryDatasetSpec_ToProto(mapCtx *direct.MapContext, in *krm.BigQueryDatasetSpec, name string) *api.Dataset {
```

Comment:
Can we cleanup the mapping function names? Currently we have `xxx_ToProto` and `xxx_ToAPI` which both referencing "google.golang.org/api/bigquery/v2". Let’s choose a consistent naming convention.